db/main/migrate/20180522000000_add_active_flag_to_crons.rb

```diff
@@ -0,0 +1,11 @@
+class AddActiveFlagToCrons < ActiveRecord::Migration[4.2]
+  self.disable_ddl_transaction!
+
+  def change
+    change_table :crons do |t|
+      t.boolean :active, default: true
+    end
+
+    add_index :crons, :next_run, where: "(active IS TRUE)", algorithm: :concurrently
```

Comment:
one thing i'm wondering is whether there is a difference between `active IS true` and `active = true`